include/acacia/mac/axapi_node.h

```diff
@@ -68,6 +68,7 @@ class AXAPINode {
    * Note: This is required for SWIG bindings to compile, but creates a node
    * which doesn't wrap an AXUIElementRef, and will cause a crash when any of
    * the accessor methods which depend on a valid AXUIElementRef are called.
+   * @ingroup axapi
```

Comment:
@spectranaut Thanks for adding these, but I'm a bit confused as to why they're needed. It seems like they mostly just add noise - the method docs are still available within the object documentation without them, but this just also bumps them up to the group.